trak.ino

```diff
@@ -27,7 +27,7 @@ void trak_move(){
   trak_enable();
   delay(200);
```

Comment:
maybe we should change this to 199

Reply:
Why tho?

Reply:
Someone tried making their first pull request :)

(with permission)